//ทำไม่ได้ เกิด error เพราะเรียกใช้ x ใน console.log(x) ก่อนที่จะประกาศตัวแปร let x = 2